src/react-components/room/MicSetupModal.js

```diff
@@ -101,18 +102,18 @@ export function MicSetupModal({
         <ToolbarButton
           icon={soundPlaying ? <VolumeHighIcon width={48} height={48} /> : <VolumeOffIcon width={48} height={48} />}
           label="Click to Test Audio"
-          preset={soundPlaying && "blue"}
+          preset={soundPlaying ? "blue" : "basic"}
           className={styles.largeToolbarButton}
+          onClick={onPlaySound}
           large
         />
       </div>
-      <SelectInputField
-        disabled={!microphoneEnabled}
-        value={selectedMicrophone}
-        options={microphoneOptions}
-        onChange={onChangeMicrophone}
-      />
-      <ToggleInput label="Mute My Microphone" value={microphoneMuted} onChange={onChangeMicrophoneMuted} />
+      {microphoneEnabled && (
+        <>
+          <SelectInputField value={selectedMicrophone} options={microphoneOptions} onChange={onChangeMicrophone} />
```

Comment:
No I think we should just change the PropType for value to `PropTypes.any`. It can be undefined, a string or a number, and with the change to `getSelectedItem`, I guess it can be one of the option objects as well.